src/software/amazon/ion/IonWriter.java

```diff
@@ -14,14 +14,15 @@
 
 package software.amazon.ion;
 
+import software.amazon.ion.system.IonTextWriterBuilder;
```

Comment:
For posterity: the Eclipse preferences stored in `.settings`. We can store preferences for IntelliJ too if needed.